ws-android-sdk/Dockerfile

```diff
@@ -0,0 +1,105 @@
+FROM cloud9/workspace
+
+# first attempt by Jeremy Ellis to add the Android_SDK
+
+#MAINTAINER Fabian Jakobs <fabian@c9.io>
+
+ADD ./files/workspace /home/ubuntu/workspace
+
+RUN chmod -R g+w /home/ubuntu/workspace && \
+    chown -R ubuntu:ubuntu /home/ubuntu/workspace
+    
+
+# Android Build Dockerfile for Parafuzo.
+#
+# This is a fork from:
+# https://github.com/lukin0110/docker-android-build
+#
+
+# Pull base image.
+#FROM ubuntu:14.04
+
+#MAINTAINER Parafuzo <dev@parafuzo.com>
+MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
+
+
+
+# Update, upgrade and install packages
+RUN \
+    apt-get update && \
+    apt-get -y install curl unzip python-software-properties software-properties-common lib32stdc++6 lib32z1
+
+# Install Oracle Java JDK
+# https://www.digitalocean.com/community/tutorials/how-to-install-java-on-ubuntu-with-apt-get
+# https://github.com/dockerfile/java/blob/master/oracle-java7/Dockerfile
+RUN \
+    echo oracle-java7-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
```

Comment:
Should this be java8 now? Since 9 is coming out soon and 7 is 'legacy' But I'm not sure where android SDK fits with java support to be honest.